wgpu-core/src/resource.rs

```diff
@@ -499,6 +538,28 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
 
         hal_surface_callback(hal_surface)
     }
+
+    /// # Safety
+    /// - The raw command encoder handle must not be manually destroyed
+    pub unsafe fn command_encoder_as_hal_mut<
```

Comment:
Do I need to add the not wasm #cfg's here, and in a few other places?

Reply:
We should